app/clients/spring_client.py

```
@@ -0,0 +1,208 @@
+    async def health_check(self) -> Dict[str, Any]:
+        """Spring Boot 헬스체크"""
+        try:
+            response = await self.client.get("/health")
+            response.raise_for_status()
+            return response.json()
+        except Exception as e:
+            logger.error(f"Spring Boot 헬스체크 실패: {e}")
+            raise
+    
+    async def send_recommendation_result(self, 
+                                       member_id: int, 
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)